readme.md

```diff
@@ -139,6 +139,16 @@
 | ------: | -------: | -----: |
 |   2 - 4 |        3 | 15-21m |
 
+### [Sorry!](https://boardgamegeek.com/boardgame/2407/sorry)
```

Comment:
It looks like [the linter failed the build](https://travis-ci.org/edm00se/awesome-board-games/builds/620768003#L239) because it picked up on the `!` at the end of the heading line.

Removing it should make that quiet down.
```suggestion
### [Sorry](https://boardgamegeek.com/boardgame/2407/sorry)
```